src/migrations.js

```diff
@@ -24,7 +21,7 @@ require.scopes.migrations = (function() {
 var exports = {};
 exports.Migrations= {
   changePrivacySettings: function() {
-    if (!chrome.extension.inIncognitoContext) {
+    if (!chrome.extension.inIncognitoContext && chrome.privacy ) {
```

Comment:
`browser.extension.inIncongnitoContext` is broken, I made a bug report [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1276025).